### PR TITLE
Add Continuous Deployment tools (`auto`, packaging workflows, zenodo)

### PR DESCRIPTION
**Pull request by smoia**

### CODE REVIEW

**Comment by rgbayrak** [comment]
What needs to be checked here? @smoia

**Comment by smoia** [comment]
If the content of the files make sense!

File: .zenodo.json
Comment (rgbayrak) [inline .zenodo.json:1]: Could include other metadata:
    "publication_date": "YYYY-MM-DD", 
    "description": "A brief description of your project",
    "communities": [{"identifier": "community_identifier"}],
    "version": "Version number (e.g., 1.0.0)",

Comment (smoia) [reply .zenodo.json:1]: These are autoatically added by the publishing bot at each release!

**Comment by smoia** [comment]
:rocket: PR was released in [`0.4.0`](https://github.com/physiopy/peakdet/releases/tag/0.4.0) :rocket:

**Pull request merged**